### ISSUE TITLE
Migrations for MongoDB

### ISSUE THREAD

**Issue by gitcommitshow** [ArekSredzki/electron-release-server#295]
I see that the latest release server has migrations only for Postgres at the moment. I propose to create similar migrations For MongoDB for users who want to upgrade from v1.3 to latest version. I would love to create a PR for the same. 
How I plan to do that:

1. [Review migrations](https://github.com/ArekSredzki/electron-release-server/tree/master/migrations) for Postgres. 
2. For MongoDB, I think only `Update` and `Constraint` related command will be relevant. So write mongodb equivalent for those lines only.
3. Figure out how to integrate these new changes in `db-migrate` in the context of mongodb.

Questions I have

1. As I see there some new collections have been added, is there any requirement of writing bootstrap data as well?
2. Any other suggestions before I jump into this?

**Comment by zaourzag** [comment]
Hi, I was looking around if I could find mongodb. Seeing that i prefer it all day any day over postgress. 
2. Prob just write it in docs that mongo is supported.

**Comment by gitcommitshow** [comment]
Yes. I think what's pending to call it mongodb supported is this issue only i.e. "writing mongodb migrations". This project uses waterline ORM so you should be able to work with all the databases that waterline supports. But as I see that the author uses postgres as their choice of db and tests the project for that only. The migrations are written for that only. So when we upgrade the project while using mongodb, we can't say for sure that the application won't break.

**Comment by gitcommitshow** [comment]
While fixing this, ran into issue #297
